consensus/parlia/parlia.go

```diff
@@ -1828,6 +1828,9 @@ func (p *Parlia) distributeIncoming(val common.Address, state *state.StateDB, he
 		}
 	}
 	log.Trace("distribute to validator contract", "block hash", header.Hash(), "amount", balance)
+
```

Comment:
add following code, and remove `balance = balance.Sub(balance, rewards)` ?
	balance := state.GetBalance(consensus.SystemAddress)
	if balance.Cmp(common.U2560) <= 0 {
		return nil
	}